source/js/modules/Modal.js

```diff
@@ -108,10 +109,11 @@ class ModalHandler {
 
   /**
      * `Show the modal-box`
+     * @param {Number} time The time for the modal box to slide down
      */
   show() {
-    if (this.sliding) $(this.view.container.parentElement).slideDown(400);
-    else $(this.view.container.parentElement).fadeIn(400);
+    // if (this.sliding) $(this.view.container.parentElement).animate({ height: '100vh' }, time);
```

Comment:
why do we keep this as comment?

Reply:
Just in case, it's our second option if the first one ducks up.

Reply:
please explain in code for future readers please. why second option? what ducks up here? Ie what are the instructions for when to uncomment this?

Reply:
Actually it doesn't need to be there, so I removed it